RevenueCat/Scripts/Purchases.cs

```diff
@@ -54,6 +54,16 @@ public partial class Purchases : MonoBehaviour
     [Header("Advanced")]
     [Tooltip("Set this property to your proxy URL before configuring Purchases *only* if you've received a proxy key value from your RevenueCat contact.")]
     public string proxyURL;
+    
+	[Header("Dangerous Settings")]
+    [Tooltip("Disable or enable subscribing to the StoreKit queue." +  
+	"If this is disabled, RevenueCat won't observe the StoreKit queue or check current purchasers, " +
+	"and it will not sync any purchase automatically. Call syncPurchases whenever a new transaction is " +
+	"completed so the receipt is sent to RevenueCat's backend. " +
+	"In iOS, consumables disappear from the receipt after the transaction is finished, so make sure purchases " +
+	"are synced before finishing any consumable transaction, otherwise RevenueCat won't register the purchase. " +
+	"Auto syncing of purchases is enabled by default.")]
+    public bool autoSyncPurchases = true;
```

Comment:
This version of Unity (3.2.0) doesn't have a runtime setup option. That is getting added with the Amazon support. Since this is a build I am building for a customer, I added it as an option in the Editor for now. 

When we release a Unity version with Amazon support this should not be available as an Editor option since it's quite a dangerous setting.